src/appConfig.js

```diff
@@ -76,7 +79,9 @@ configuration.queryparams =
         'GageStatsServicesStatGroups': '/statisticgroups/',
         'GageStatsServicesNearest': '/stations/Nearest?lat={0}&lon={1}&radius={2}&geojson=false&includeStats=true',
         'GageStatsServicesNetwork': '/stations/Network?lat={0}&lon={1}&distance={2}&includeStats=true&geojson=false',
-        'GageStatsServicesBounds': '/stations/Bounds?xmin={0}&xmax={1}&ymin={2}&ymax={3}&geojson=true'
+        'GageStatsServicesBounds': '/stations/Bounds?xmin={0}&xmax={1}&ymin={2}&ymax={3}&geojson=true',
+        'FlowAnywhereEstimates': '/models/FLA/estimate?state=IA',
```

Comment:
This could be '/models/FLA/estimate?state={0}' in case it ever gets added to other states.